lib/pages/projects/project_widget.dart

```diff
@@ -70,27 +69,43 @@ class ProjectRow extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     HomeBloc homeBloc = BlocProvider.of(context);
-    return ListTile(
-      key: ValueKey("tile_${project.name}_${project.id}"),
-      onTap: () {
-        homeBloc.applyFilter(project.name, Filter.byProject(project.id!));
-        context.safePop();
+    return Dismissible(
```

Comment:
Test for dismissing widget to delete project

Reply:
Test case `Swipe left to delete project` is covering this.